src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -1,19 +1,68 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-const ProfileHeader = () => (
-  <div className="vads-u-background-color--primary-darkest vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3">
-    <div className="row">
-      <h1 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
-        Your Profile
-      </h1>
-      <h2 className="vads-u-font-size--h3">
-        Kimberly Elizabeth Smith Washington
-      </h2>
-      <h3 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
-        United States Army Reserve
-      </h3>
+const ProfileHeader = ({
+  userFullName: { first, middle, last },
+  latestBranchOfService,
+}) => {
+  const imagePaths = {
+    A: '/img/vic-army-symbol.png',
+    C: '/img/vic-cg-emblem.png',
+    F: '/img/vic-air-force-coat-of-arms.png',
+    N: '/img/vic-navy-emblem.png',
+    M: '/img/vic-usmc-emblem.png',
+    VASeal: '/img/vic-va-seal.png',
+  };
+
+  return (
+    <div className="headerWrapper vads-u-background-color--gray-dark vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3 vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center">
+      <div className="headerContentWrapper usa-grid usa-grid-full vads-u-display--flex vads-u-align-items--center">
+        <div className="vads-u-padding-x--6 usa-width-one-fourth">
```

Comment:
we should support hiding the badge entirely if we don't have the `latestBranchOfService`